### ISSUE TITLE
how to understand the get_pos_logits() in the class ProtoLoss

### ISSUE THREAD

**Issue by PhDCJojo** [korawat-tanwisuth/Proto_DA#1]
Dear author: Thank you very much for sharing this code. During learning this code, I can not find the explanation about the function get_pos_logits() in Equation 8 of your paper. Can you tell me what this function does？
https://github.com/korawat-tanwisuth/Proto_DA/blob/332e6ed5814db98d33cd92842012e57298b631fb/dalib/adaptation/proto.py#L40
 
Best wishes.

**Comment by korawat-tanwisuth** [comment]
Thanks for your interest in our work. This function computes the logit after the proportion has been updated with exponential moving average. This logit is used in the computation of the bidirectional transport loss in the paper.
